components/HeroBlockTechnology/HeroBlockTechnology.scss

```diff
@@ -120,23 +120,21 @@
 
       @include tablet-xl {
         left: 0;
-        bottom: -507px;
-        width: 290px;
-        height: 985px;
+        top: 144px;
+        width: auto;
+        height: calc(100% - 144px);
+        max-width: 750px;
```

Comment:
the picture doesn't stretch to 750, please check.
I also didn't see 750 in the design.